Examples/ObjectiveC/ShapeAnnotationsExample.m

```diff
@@ -0,0 +1,85 @@
+#import "ShapeAnnotationsExample.h"
+@import Mapbox;
+
+NSString *const MBXExampleShapeAnnotations = @"ShapeAnnotationsExample";
+
+@interface ShapeAnnotationsExample () <MGLMapViewDelegate>
+
+@end
+
+@implementation ShapeAnnotationsExample
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+
+    CLLocationCoordinate2D pointCoordinate = CLLocationCoordinate2DMake(45.531, -122.666);
+
+    // Since structs aren't supported in Objective-C arrays,
+    // store coordinates in a compatible C++ array instead.
+    CLLocationCoordinate2D lineCoordinates[] = {
+        CLLocationCoordinate2DMake(45.526, -122.677),
+        CLLocationCoordinate2DMake(45.523, -122.677),
+        CLLocationCoordinate2DMake(45.523, -122.674),
+        CLLocationCoordinate2DMake(45.522, -122.674),
+        CLLocationCoordinate2DMake(45.518, -122.676),
+        CLLocationCoordinate2DMake(45.517, -122.673),
+        CLLocationCoordinate2DMake(45.513, -122.676)
+    };
+
+    NSUInteger numberOfLineCoordinates = sizeof(lineCoordinates) / sizeof(CLLocationCoordinate2D);
+
+    CLLocationCoordinate2D polygonCoordinates[] = {
+        CLLocationCoordinate2DMake(45.512, -122.663),
+        CLLocationCoordinate2DMake(45.512, -122.654),
+        CLLocationCoordinate2DMake(45.522, -122.654),
+        CLLocationCoordinate2DMake(45.522, -122.663),
+        CLLocationCoordinate2DMake(45.512, -122.663)
+    };
```

Comment:
Putting these all in `viewDidLoad` (before the map view is set up, even) feels a bit unorganized. Could these be broken out into properties/methods?